site/docs/spark-procedures.md

```diff
@@ -209,16 +209,16 @@ Erase all snapshots older than the current timestamp but retain the last 5 snaps
 CALL hive_prod.system.expire_snapshots(table => 'db.sample', older_than => now(), retain_last => 5)
 ```
 
-### `remove_orphan_files`
+### `delete_orphan_files`
```

Comment:
Action was called DeleteOrphanFiles but procedure was called remove_orphan_files. 
Hence changed the procedure name as well to match the action name.